modules/visual_script/doc_classes/VisualScriptBuiltinFunc.xml

```diff
@@ -80,7 +80,7 @@
 			Return the natural logarithm of the input. Note that this is not the typical base-10 logarithm function calculators use.
 		</constant>
 		<constant name="MATH_EXP" value="20" enum="BuiltinFunc">
-			Return [b]e[/b] raised to the power of the input. [b]e[/b] sometimes called "Euler's number" is a mathematical constant whose value is approximately 2.71828.
+			Return the mathematical constant [b]e[/b] raised to the power of the input. [b]e[/b] has an approximate value of 2.71828.
```

Comment:
I know this is existing text, but I think it would be better as "raised to the specified power" rather than "raised to the power of the input".

Reply:
Would probably need to adjust wording on other descriptions in the VisualScriptBuiltinFunc file for consistency, there are several instances of "of the input" wordings...